data/wanderer/wanderers start.txt

```diff
@@ -984,10 +991,11 @@ mission "Wanderers: Alpha Surveillance A"
 		dialog phrase "generic waypoint on visit"
 	on complete
 		log "Planted some surveillance devices, designed by the Wanderers, in Unfettered Hai space. A month from now, they will be done collecting data on ship movements in the area, and may provide a hint of where the Alphas who are selling jump drives to the Unfettered are coming from."
+		log "People" "Hiyamaharu" `Hiyamaharu is an aged Hai ambassador. He appears to have taken of Sayari's role in dealing with humans since her departure from Hai space.`
```

Comment:
```suggestion
		log "People" "Hiyamaharu" `Hiyamaharu is an aged Hai ambassador. He appears to have taken up Sayari's role in dealing with humans since her departure from Hai space.`
```